### DISCUSSION TITLE
Multisite - Users Permissions - Users should only be able to delete users from their site

### DISCUSSION THREAD

**Discussion by dean-yump** [craftcms/cms#9923] [Ideas] [answered]
Unless I am missing something, it seems like any user with admin rights can either have the permissions to delete, adnimistrate and impersonate users _from any site_ either on or off. 

![image](https://user-images.githubusercontent.com/16073493/135820244-84876e1e-1a5c-49ad-8c5c-9272d51aeac7.png)

What we are looking for is to be able to assign a user to a user group that can only administrate users on a site by site basis. So similar to the way it works for assigning user groups and creating users. We don't want admins from one site being able to suspend, delete etc users from another site.

**Comment by brandonkelly** [comment, marked as answer]
This is working as expected; users don’t belong to a site(s) in Craft; they are centrally managed. So I’ve converted this to a discussion (feature request).
